### ISSUE TITLE
merge params treatment_var (in QC section) and design (in DEseq section)?

### ISSUE THREAD

**Issue by cookkate** [R-ODAF/R-ODAF_Health_Canada#108]
These seem similar enough that they could be merged.

**Comment by mattjmeier** [comment]
Yeah I think that is logical because then it forces some level of consistency over what's shown in the QC report in terms of groupings, vs what gets analyzed together.

Originally, I had intended a bit of flexibility in what's plotted, but I think this approach makes more sense, let's do it!

**Comment by mattjmeier** [comment]
Ah so I started looking at this one.

The QC report makes plots that facet on treatment_var, which might be say, chemical. So you get one plot per chemical.

This doesn't work when you try to make it the same as design, because that would be like a "group" column for example, where each facet would end up being chemical + dose. And we probably don't want that many plots for the types of things that treatment_var is used for.

There might be a way to consolidate something in there though. treatment_var is kind of part of exp_groups sometimes... But those are also still used in a distinct way for some plots.

**Comment by mattjmeier** [comment]
Hmm actually I'm going to reopen this and think about combining it with the display facet. That is basically what we're driving at there.
